edx_proctoring/api.py

```diff
@@ -648,6 +647,12 @@ def get_exam_attempt_data(exam_id, attempt_id, is_learning_mfe=False):
         ),
     }
 
+    if attempt['status'] == ProctoredExamStudentAttemptStatus.ready_to_submit:
+        can_continue = _does_time_remain(attempt)
+        if exam['is_proctored']:
```

Comment:
One minor suggestion I would rewrite the code a little bit since we are on py3.8 already )))
```suggestion
        if (can_continue := _does_time_remain(attempt)) and exam['s_proctored']:
            can_continue = provider and not provider.should_block_access_to_exam_material()
```
with such can_continue definition L651 could be removed